### PR TITLE
Release/ea3

### PR DESCRIPTION
**Pull request by twosdai**
EA3 release changes

### CODE REVIEW

File: dockerFiles/marklogic-server-centos:base [hunk 1]
```diff
@@ -46,7 +46,9 @@ ENV MARKLOGIC_INSTALL_DIR=/opt/MarkLogic  \
     MARKLOGIC_VERSION="${ML_VERSION}" \
     MARKLOGIC_BOOTSTRAP_HOST=bootstrap \
     MARKLOGIC_ADMIN_USERNAME_FILE=mldb_admin_user \
-    MARKLOGIC_ADMIN_PASSWORD_FILE=mldb_password_user
+    MARKLOGIC_ADMIN_PASSWORD_FILE=mldb_password_user \
+    MARKLOGIC_LICENSE=mldb_liense \
```
Comment (jkerr5) [inline dockerFiles/marklogic-server-centos:base:50]: I would recommend this be "LICENSE_KEY" to match the wording in the admin UI as well as the docs https://docs.marklogic.com/REST/POST/admin/v1/init

File: src/centos/scripts/start-marklogic.sh [hunk 2]
```diff
@@ -64,7 +64,12 @@ then
 echo "MARKLOGIC_INIT is not defined, no bootstrap"
 else
 echo "MARKLOGIC_INIT is defined, bootstrapping"
-curl -X POST -d "" http://$HOSTNAME:8001/admin/v1/init
+
+curl --anyauth -i -X POST \
+   -H "Content-type:application/x-www-form-urlencoded" \
+   --data-urlencode $MARKLOGIC_LICENSE \
```
Comment (jkerr5) [inline src/centos/scripts/start-marklogic.sh:70]: This needs to specify the parameter name: "license-key=$MARKLOGIC_LICENSE"

File: src/centos/scripts/start-marklogic.sh [hunk 3]
```diff
@@ -64,7 +64,12 @@ then
...
+   --data-urlencode $MARKLOGIC_LICENSEE \
```
Comment (jkerr5) [inline src/centos/scripts/start-marklogic.sh:71]: This needs to specify the parameter name: "licensee=$MARKLOGIC_LICENSEE"

File: docker-compose/marklogic-cluster-centos.yml [hunk 4]
```diff
@@ -42,7 +42,7 @@ services:
       networks:
       - external_net
     node3:
-      image: store/marklogicdb/marklogic-server:10.0-7.3-centos-1.0.0-ea
+      image: store/marklogicdb/marklogic-server:10.0-8.1-centos-1.0.0-ea2
```
Comment (jkerr5) [inline docker-compose/marklogic-cluster-centos.yml:45]: We will update this to 10.0-8.3, right?

**Review by jkerr5 — CHANGES_REQUESTED** [review]
Main issue is that I think we need to fix the way the license key and licensee are passed in to the init call. See https://docs.marklogic.com/REST/POST/admin/v1/init

File: dockerFiles/marklogic-server-centos:base (hunk 1 above)
Comment (twosdai) [reply dockerFiles/marklogic-server-centos:base]: Ok I've updated it. 
aeecdd9

I also changed the licensee to be `LICENSEE` instead to be inline with the rest api doc.

File: src/centos/scripts/start-marklogic.sh (hunk 2 above)
Comment (twosdai) [reply src/centos/scripts/start-marklogic.sh]: Updated: aeecdd9

File: docker-compose/marklogic-cluster-centos.yml (hunk 4 above)
Comment (twosdai) [reply docker-compose/marklogic-cluster-centos.yml:45]: Yeah I just updated it: aeecdd9

Good catch.

File: src/centos/scripts/start-marklogic.sh [hunk 5]
```diff
@@ -67,8 +67,8 @@ echo "MARKLOGIC_INIT is defined, bootstrapping"
 
 curl --anyauth -i -X POST \
    -H "Content-type:application/x-www-form-urlencoded" \
-   --data-urlencode $MARKLOGIC_LICENSE \
-   --data-urlencode $MARKLOGIC_LICENSEE \
+   --data-urlencode $license-key=$LICENSE_KEY \
```
Comment (jkerr5) [inline src/centos/scripts/start-marklogic.sh:70]: Is there a reason there is a $ in front of license-key? It should be "license-key=$LICENSE_KEY" (include the quotes too)

File: src/centos/scripts/start-marklogic.sh (hunk 3 above)
Comment (twosdai) [reply src/centos/scripts/start-marklogic.sh]: Updated: aeecdd9

File: src/centos/scripts/start-marklogic.sh (hunk 5 above)
Comment (twosdai) [reply src/centos/scripts/start-marklogic.sh]: Yeah you're correct, I just tested the curl request and it fails with a parameter unauthorized error when the`$` is used. 

dd2cf38 
Updated.

**Review by vitalykorolev — APPROVED** [review]

**Pull request merged**